proxyPractice.py

```diff
@@ -0,0 +1,58 @@
+import socket, sys
+from _thread import *
+print("hi")
+host = ''
```

Comment:
You want the host to be your local machine hostname
`host = socket.gethostname() # Get local machine name`